.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+
+# Cache pip downloads
+cache:
+    # Apparently if you override the install command that silently disables the
+    # cache: pip support. This is less than ideal and @dstufft opened up
+    # travis-ci/travis-ci#3239 to hopefully get that addressed. For now we
+    # manually add the pip cache directory to the build cache.
+    # (from django-oscar/.travis.yml)
+    directories:
+      - ~/.cache/pip
```

Comment:
> Frankly, this is such a simple project, I wouldn't bother with either.

Agreed. I might just remove it, it's not that necessary on a small project. I prefer to keep things simple.